<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="PINKYPRIYA" ignored="1" name="Default test(failed)" tests="1" failures="1" timestamp="21 Dec 2020 06:42:51 GMT" time="0.158" errors="0">
  <testcase name="test2" time="0.073" classname="org.test.FailedCaseManual">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [priya] but found [Priya]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [priya] but found [Priya]
at org.test.FailedCaseManual.test2(FailedCaseManual.java:19)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase name="test1" time="0.0" classname="org.test.FailedCaseManual">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
